### PR TITLE
Sdk 68

### PR DESCRIPTION
**Pull request by ankur-dwivedi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,45 @@ ImageKit Angular SDK allows you to resize, optimize, deliver, and upload images
 
 ImageKit is complete media storage, optimization, and transformation solution that comes with an image and video CDN. It can be integrated with your existing infrastructure - storage like AWS S3, web servers, your CDN, and custom domain names, allowing you to deliver optimized images in minutes with minimal code changes.
 
+## Breaking changes - Upgrading from 2.x to 3.x version
+3.x version has breaking changes as listed below.
+* Previously, when using this SDK, we had to pass `authenticationEndpoint` which is used by SDK internally for fetching security parameters i.e `signature`, `token`, and `expire`.
```
Comment (imagekitio) [inline README.md:16]: This point is not needed, we have mentioned everything in the next.

File: sdk/lib/src/imagekitio-angular/ik-upload/ik-upload.component.ts [hunk 2]
```diff
@@ -45,6 +46,13 @@ export class IkUploadComponent implements AfterViewInit {
     this.buttonRef && this.buttonRef.addEventListener('click', ()=>{this.el.nativeElement.children[0].click()});
   }
 
+  abort() {
+    if (this.xhr) {
+      this.xhr.abort();
+      console.log('Upload aborted');
```
Comment (imagekitio) [inline sdk/lib/src/imagekitio-angular/ik-upload/ik-upload.component.ts:52]: remove console log added during development/debugging.

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: npm install, build, and test
       run: |
-        npm install -g @angular/cli@1.4.10
+        npm install -g @angular/cli@1.7.4
```
Comment (imagekitio) [inline .github/workflows/nodejs.yml:22]: Why is this changed and what is the impact of this for user?

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ankur-dwivedi) [reply README.md]: done removed the first point in [8887b1a](https://github.com/imagekit-developer/imagekit-angular/pull/92/commits/8887b1a64d7edbc8e8fb4fccb37417dce7656d75)

Comment (ankur-dwivedi) [reply README.md]: done removed the log in [8887b1a](https://github.com/imagekit-developer/imagekit-angular/pull/92/commits/8887b1a64d7edbc8e8fb4fccb37417dce7656d75)

File: sdk/lib/src/imagekitio-angular/ik-upload/ik-upload.component.ts (hunk 2 above)
Comment (ankur-dwivedi) [reply sdk/lib/src/imagekitio-angular/ik-upload/ik-upload.component.ts]: done removed the log in [8887b1a](https://github.com/imagekit-developer/imagekit-angular/pull/92/commits/8887b1a64d7edbc8e8fb4fccb37417dce7656d75)

File: .github/workflows/nodejs.yml (hunk 3 above)
Comment (ankur-dwivedi) [reply .github/workflows/nodejs.yml]: I have reverted this change.  This change was made as there was an open Mr by GitHub bot to upgrade the dependency `@angular/cli@1.4.10`. On testing with upgraded ci SDK, it worked fine, but this change could have restricted users with `@angular/cli@1.7.4`.
